incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -170,4 +170,20 @@ var _ = Describe("Issues", func() {
 		// test: delete namespace 
 		mustRun("kubectl delete ns", nsChild)
 	})
+
+	It("Should not delete parent namespace without setting allowCascadingDelete -issue #716", func(){
```

Comment:
We may need more context when these test cases are all put in one e2e test for users to run.
r/Should not delete a parent of a **subnamespace** if allowCascadingDelete is not set.

Reply:
done